util/chunk/chunk_util.go

```diff
@@ -15,6 +15,51 @@ package chunk
 
 import "github.com/pingcap/errors"
 
+// CopySelectedJoinRowsDirect directly copies the selected joined rows from the source Chunk
+// to the destination Chunk.
+// Return true if at least one joined row was selected.
+func CopySelectedJoinRowsDirect(src *Chunk, innerColOffset, outerColOffset int, selected []bool, dst *Chunk) (bool, error) {
```

Comment:
`innerColOffset` and `outerColOffset` seems to be useless.

Reply:
removed.